etc/config/compiler-explorer.defaults.properties

```diff
@@ -32,3 +32,4 @@ privacyPolicyEnabled=false
 
 supportsLibraryCodeFilter=false
 ldPath=${exePath}/../lib|${exePath}/../lib32|${exePath}/../lib64
+ceToolsPath=../compiler-explorer-path
```

Comment:
this should say ../compiler-explorer-tools maybe?

Reply:
Good catch :)